drf_link_navigation_pagination/__init__.py

```diff
@@ -100,9 +102,23 @@ def _get_updated_url(url: str, domain: str) -> str:
     return new_parse_result.geturl()
 
 
-def _add_request_path(url: str, request_path: str) -> str:
+def _add_request_path(url: str, request_path: str, has_custom_domain: bool = False) -> str:
+    """"
+    This function adds a request path to a URL.
+    If a custom domain is also being used, the trailing slash is removed from the original path.
+    """
     parse_result = urlparse(url)
-    final_path = _urljoin(request_path, parse_result.path)
+    original_path = parse_result.path
+    
+    has_trailing_slash = original_path.endswith("/")
+    
+    final_path = _urljoin(request_path, original_path)
+    
+    if not has_custom_domain and has_trailing_slash:
+        final_path = final_path.rstrip("/") + "/"
+    else:
+        final_path = final_path.rstrip("/")
```

Comment:
[suggestion]
If the strip always happens why not?


```suggestion
	final_path = final_path.rstrip("/")
    if not has_custom_domain and has_trailing_slash:
        final_path = final_path + "/"
```

Reply:
> If the strip always happens why not?

Sorry, couldn't understand you point here.
Right now, the lib is removing the trailing slash, and this breaks the pagination for endpoints that requires a trailing slash.

Reply:
My point was just that an else is not necessary, since the strip happens on each path, you could simplify and just append the slash